sympy/functions/special/gamma_functions.py

```diff
@@ -1009,3 +1009,67 @@ def trigamma(x):
     .. [3] http://functions.wolfram.com/GammaBetaErf/PolyGamma2/
     """
     return polygamma(1, x)
+
+def multivariate_gamma(x, p):
+    r"""
+    The multivariate gamma function is a generalization of the gamma function i.e,
+
+    .. math::
+        \Gamma_p(z) = \pi^{p(p-1)/4}\prod_{k=1}^p \Gamma[z + (1 - k)/2].
+
+    Special case, multivariate_gamma(x, 1) = gamma(x)
+
+    Parameter
+    =========
+    p: order or dimension of the multivariate gamma function
+
+    Examples
+    ========
+
+    >>> from sympy import S, I, pi, oo, gamma, multivariate_gamma
+    >>> from sympy import Symbol
+    >>> x = Symbol('x')
+    >>> p = Symbol('p', positive=True, integer=True)
+
+    >>> multivariate_gamma(x, p)
+    pi**(p*(p - 1)/4)*Product(gamma(-k/2 + x + 1/2), (k, 1, p))
+
+    Several special values are known:
+    >>> multivariate_gamma(1, 1).doit()
+    1
+    >>> multivariate_gamma(4, 1).doit()
+    6
+    >>> multivariate_gamma(S(3)/2, 1).doit()
+    sqrt(pi)/2
+
+    Writing multivariate_gamma in terms of gamma function
+    >>> multivariate_gamma(x, 1).doit() == gamma(x)
+    True
+    >>> multivariate_gamma(x, 2).doit() == pi**(0.5)*gamma(x)*gamma(x - 0.5)
+    True
+    >>> multivariate_gamma(x, 3).doit() == pi**(1.5)*gamma(x)*gamma(x - 0.5)*gamma(x - 1)
+    True
```

Comment:
Can these examples be written as the following?
```diff
-    >>> multivariate_gamma(x, 1).doit() == gamma(x)
+    >>> multivariate_gamma(x, 1).doit()
+    gamma(x)
```

Reply:
Done. Thanks!